src/main/resources/META-INF/plugin.xml

```diff
@@ -53,6 +53,6 @@
       language="Earthly"
       patterns="Earthfile;*.earth;*.earthly;*.Earthfile"/>
     <editorHighlighterProvider filetype="Earthfile" implementationClass="dev.earthly.plugin.language.syntax.highlighting.EarthlyEditorHighlighterProvider"/>
-    <lang.commenter language="Earthly" implementationClass="dev.earthly.plugin.language.syntax.highlighting.EarthlyCommenter"/>
```

Comment:
Changed the package since that class file seem to change case, and git doesn't like that.
Besides, language package seems more appropriate based on intellij _Comenter_ package